long-term-storage.md

```diff
@@ -59,61 +60,69 @@ $ helm install kubecost/cost-analyzer \
+![image](https://raw.githubusercontent.com/kubecost/docs/main/images/thanos-query.png)
 
 If you navigate to the *Stores* using the top navigation bar, you should be able to see the status of both the `thanos-store` and `thanos-sidecar` which accompanied prometheus server:
 
-![image](https://user-images.githubusercontent.com/334480/66617048-58960700-eba1-11e9-9f68-d007fcb11410.png)
+![image](https://raw.githubusercontent.com/kubecost/docs/main/images/thanos-store.png)
 
 Also note that the sidecar should identify with the unique `cluster_id` provided in your values.yaml in the previous step. Default value is `cluster-one`.
 
-
 The default retention period for when data is moved into the object storage is currently *2h* - This configuration is based on Thanos suggested values. __By default, it will be 2 hours before data is written to the provided bucket.__
 
 Instead of waiting *2h* to ensure that thanos was configured correctly, the default log level for the thanos workloads is `debug` (it's very light logging even on debug). You can get logs for the `thanos-sidecar`, which is part of the `prometheus-server` pod, and `thanos-store`. The logs should give you a clear indication of whether or not there was a problem consuming the secret and what the issue is. For more on Thanos architecture, view [this resource](https://github.com/thanos-io/thanos/blob/master/docs/design.md).
 
 ### Troubleshooting
 
 #### Cluster not writing data to thanos bucket
+
 If a cluster is not successfully writing data to the bucket, we recommend reviewing `thanos-sidecar` logs with the following command:
 
-```
+```shell
 kubectl logs kubecost-prometheus-server-<your-pod-id> -n kubecost -c thanos-sidecar
 ```
 
 Logs in the following format are evidence of a successful bucket write:
 
-```
+```text
 level=debug ts=2019-12-20T20:38:32.288251067Z caller=objstore.go:91 msg="uploaded file" from=/data/thanos/upload/01KL5YG9CQYZ81G9BZMTM3GJFH/meta.json dst=debug/metas/01KL5YG9CQYZ81G9BZMTM3GJFH.json bucket=kc-thanos
-
 ```
 
-#### Stores not listed at the /stores endpoint
-If thanos-query can't connect to both the sidecar and the store, you may want to directly specify the store GRPC service address instead of using DNS discovery (the default).
-You can quickly test if this is the issue by running          
+#### Stores not listed at the `/stores` endpoint
+
+If thanos-query can't connect to both the sidecar and the store, you may want to directly specify the store gRPC service address instead of using DNS discovery (the default). You can quickly test if this is the issue by running
+
  `kubectl edit deployment kubecost-thanos-query -n kubecost`
- and adding 
- `- --store=kubecost-thanos-store-grpc.kubecost:10901` to the container args. This will cause a query restart and you can visit /stores again to see if the store has been added.
+
+and adding
+
+`- --store=kubecost-thanos-store-grpc.kubecost:10901`
```

Comment:
Is this first dash actually needed? Just want to confirm.

May need to ask @AjayTripathy if not easy to answer.

Reply:
Excellent question, I'm not entirely sure, I just played with the formatting a bit to make it easier to comprehend.

Reply:
I strongly suspect that this should be:
```suggestion
`--store=kubecost-thanos-store-grpc.kubecost:10901`
```

I'm fine with making that change and then merging. Ajay can always circle back here if there is actually some very quixotic behavior in Thanos's args.